src/templates/components/CommunityTemplateInstallOverlay.tsx

```diff
@@ -128,29 +128,48 @@ class CommunityTemplateInstallOverlayUnconnected extends PureComponent<Props> {
         this.props.resourcesToSkip,
         this.props.stagedTemplateEnvReferences
       )
-      event('template_install', {templateUrl: this.props.stagedTemplateUrl})
+      event('template_install.success', {
+        templateUrl: this.props.stagedTemplateUrl,
+      })
     } catch (err) {
+      event('template_install.failure', {
+        templateUrl: this.props.stagedTemplateUrl,
+      })
       this.props.notify(communityTemplateInstallFailed())
       reportErrorThroughHoneyBadger(err, {
         name: 'Failed to install community template',
       })
       return
     }
+    const templateDetails = getTemplateNameFromUrl(this.props.stagedTemplateUrl)
+    const templateUrl = this.props.stagedTemplateUrl
```

Comment:
we want to keep the `getTemplateNameFromUrl` call inside the `try` since it can error

```suggestion
    let templateDetails
    let templateUrl
```

then in the try

```ts
try {
  templateDetails = getTemplateNameFromUrl(this.props.stagedTemplateUrl)
  templateUrl = this.props.stagedTemplateUrl
```

Reply:
I've checked the `getTemplateNameFromUrl` before moving this out of the try block and there are just a couple of `String.split` and `Array.pop` calls inside, and those don't seem to throw an error. But for sure, having it inside the try block is 100% secure.